src/values.js

```diff
@@ -99,7 +92,9 @@ Ref.prototype.toJSON = function() {
 
 /** @ignore */
 Ref.prototype.toString = function() {
-  return this.value;
+  var cls = this.class !== undefined ? ', class=' + this.class.toString() : '';
+  var db = this.database !== undefined ? ' database=' +  this.database.toString() : '';
+  return 'Ref(id=' + this.id + cls + db + ')';
```

Comment:
I just realized this strings are a big part of our dashboard repl. People can right now just copy and past references' representations back into the repr and run them. I think we should keep this behaviour by making the string representation of a Ref type a valid Ref constructor.

Reply:
this will be tricky, wouldn't be easier to the dashboard manage this?

Reply:
That's a good point. I'll open a ticket for the dashboard work.